### PR TITLE
protobuf: Add Message.FromString static method.

### PR DESCRIPTION
**Pull request by benjaminp**

### CODE REVIEW

File: third_party/2and3/google/protobuf/message.pyi [hunk 1]
```diff
@@ -31,6 +33,8 @@ class Message:
     def HasExtension(self, extension_handle): ...
     def ClearExtension(self, extension_handle): ...
     def ByteSize(self) -> int: ...
+    @classmethod
+    def FromString(cls : Type[_T], s: Any) -> T
```
Comment (srittau) [inline third_party/2and3/google/protobuf/message.pyi]: Missing `: ...`

**Comment by srittau** [comment]
I can't see `Message.FromString` in the [current version of the file](https://github.com/protocolbuffers/protobuf/blob/master/python/google/protobuf/message.py), at least not in the abc. Where is it defined?

**Comment by benjaminp** [comment]
It's magically added to generated code: https://github.com/protocolbuffers/protobuf/blob/7efcc0466c31e18842d612b1a70cec3f1c9bfb23/python/google/protobuf/internal/python_message.py#L782

**Comment by benjaminp** [comment]
(Documentation here: https://developers.google.com/protocol-buffers/docs/reference/python-generated#message)

**Pull request merged**